src/Service/Resolvers/SqlPaginationUtil.cs

```diff
@@ -63,12 +116,12 @@ public static JsonDocument CreatePaginationConnectionFromJsonElement(JsonElement
                 {
                     // use rootEnumerated to make the *Connection.items since the last element of rootEnumerated
                     // is removed if the result has an extra element
-                    connectionJson.Add(QueryBuilder.PAGINATION_FIELD_NAME, JsonSerializer.Serialize(rootEnumerated.ToArray()));
+                    connection.Add(QueryBuilder.PAGINATION_FIELD_NAME, JsonSerializer.Serialize(rootEnumerated.ToArray()));
```

Comment:
This is could also be removed `JsonSerializer.Serialize(rootEnumerated.ToArray())` in a future PR